lib/upsert/merge_function.rb

```diff
@@ -30,13 +30,15 @@ def unique_name(table_name, selector_keys, setter_keys)
     attr_reader :controller
     attr_reader :selector_keys
     attr_reader :setter_keys
+    attr_reader :disable_native
 
-    def initialize(controller, selector_keys, setter_keys, assume_function_exists)
+    def initialize(controller, selector_keys, setter_keys, assume_function_exists, disable_native)
       @controller = controller
       @selector_keys = selector_keys
       @setter_keys = setter_keys
       validate!
-      create! unless assume_function_exists
+      @disable_native = disable_native
+      create! unless assume_function_exists || @disable_native == false
```

Comment:
Need to explicitly check `== false` because if `@disable_native` is nil then we'll still create the function if necessary?